tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2334,6 +2369,9 @@ def test_tiny_token_timestamp_batch_generation(self):
 
     @slow
     def test_tiny_token_timestamp_generation_longform(self):
+        # modified here since we add eos token + start tokens in generate
+        # nevertheless would have to be tested and compared with expected from openai 
+
```

Comment:
Not sure to follow what changed

Reply:
The add 0.0000s and 82.6000 correspond respectively to the `decoder_input_ids` and `eos` tokens that I've added in long form generation. Yet as discussed [here](https://github.com/huggingface/transformers/pull/34135#issuecomment-2410846832), this is likely to be removed and therefore this will be reverted to the original value.